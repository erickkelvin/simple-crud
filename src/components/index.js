import List from './List/List.js';
import ListItem from './ListItem/ListItem.js';
import Form from './Form/Form.js';
import Field from './Field/Field.js';
import Button from './Button/Button.js';
import Spinner from './Spinner/Spinner.js';
import NotFound from './NotFound/NotFound.js';

export {
  List,
  ListItem,
  Form,
  Field,
  Button,
  Spinner,
  NotFound
};