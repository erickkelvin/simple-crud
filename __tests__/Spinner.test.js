import { Spinner } from 'components';

describe('Spinner test', () => {
  const spinner = new Spinner();

  it('render', () => {
    expect(spinner.render()).toMatchSnapshot();
  });
});