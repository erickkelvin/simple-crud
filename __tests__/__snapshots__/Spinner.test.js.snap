// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner test render 1`] = `
"
      <div class='spinner'><div></div><div></div><div></div><div></div></div>
    "
`;
